pkg/operator/apis/monitoring/v1/pod_config.go

```diff
@@ -382,6 +386,76 @@ func relabelingsForMetadata(keys map[string]struct{}) (res []*relabel.Config) {
 			TargetLabel:  "node",
 		})
 	}
+	if _, ok := keys["top_level_controller_name"]; ok {
+		res = append(res,
+			// Default to pod name
```

Comment:
nit: Can we use full sentence and trailing `.` for comments in this file, for clarity? thanks!